engine/src/main/java/com/datatorrent/stram/util/SecurityUtils.java

```diff
@@ -87,9 +105,59 @@ public static boolean isHadoopWebSecurityEnabled()
     return hadoopWebSecurityEnabled;
   }
 
-  public static boolean isStramWebSecurityEnabled()
+  @VisibleForTesting
+  protected static boolean isStramWebSecurityEnabled()
   {
     return stramWebSecurityEnabled;
   }
 
+  /**
+   * Setup security related configuration for {@link org.apache.hadoop.yarn.webapp.WebApp}.
+   * @param config
+   * @param sslConfig
+   * @return
+   */
+  public static Configuration configureWebAppSecurity(Configuration config, SSLConfig sslConfig)
```

Comment:
I guess you would like to leave the name as is and not change it to build. I will merge it soon.